lib/ApiMuncherWrapper.rb

```diff
@@ -1,23 +1,21 @@
 require 'httparty'
 # did you change this ? Then restart the server!
 class ApiMuncherWrapper
+
   BASE_URL = "https://api.edamam.com/search"
   # should this be listed as Token/ id instead/ 
   APP_ID = ENV['app_id']
   APP_KEY = ENV['app_key']
   
-  # https://developer.edamam.com/edamam-docs-recipe-api
-  # from Edamam q = query text, r = return of recipe based on it's id
-  # attr_reader :recipe, :q, :url, :image, :returns, :r
-  # def initialize(recipe, q, url, image, returns, r)
-  #   @recipe = recipe
-  #   @q = q
-  #   @url = url
-  #   @image = image
-  # end
-
-  def self.get_recipe
-    url = BASE_URL + "?q=bacon&from=0&to=10" + "&app_id=#{APP_ID}" + "&app_key=#{APP_KEY}"
+
+  def self.get_recipes(query, from=nil)
+
+    url = BASE_URL + "?q=#{query}" + "&app_id=#{APP_ID}" + "&app_key=#{APP_KEY}" + "#{from}"
+
+    puts "*******" + url + "*******"
+    
+    # url = BASE_URL + "?q=bacon&from=0&to=10" + "&app_id=#{APP_ID}" + "&app_key=#{APP_KEY}"
+
     data = HTTParty.get(url)
     # return data['hits'][0]['recipe']['label']
     recipes = []
```

Comment:
You might want to put in some checks in for if the API responds with something unexpected, like if 'hits' is null.  

I would also probably setup a recipe class and return either a specific recipe or an array of them.  That way you would separate the implementation of the Recipe API from it's use.  It would let you switch APIs or other sources of the recipes easier.